Makefile

```diff
@@ -62,19 +62,27 @@ work/imagenet/parsed_bb.csv: work/.imagenet_training_bbox_sentinel \
 	python motion_tracker/data_setup/parse_imagenet_bb.py data/imagenet/bb/ \
 		work/imagenet/parsed_bb.csv
 
+# Run after images are downloaded, to filter out those images 
```

Comment:
yeah. I'd just remove the comment.  At first, it made me wonder if there's something more than meets the eye (e.g. a dependency we couldn't represent in the Makefile).